src/main/java/Duke.java

```diff
@@ -1,10 +1,197 @@
+import duke.Deadline;
+import duke.Event;
+import duke.Task;
+import duke.ToDo;
+
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+
 public class Duke {
+
+    public static final String BYE = "bye";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        ArrayList<Task> taskLists = new ArrayList<Task>();
+
+
```

Comment:
Avoid long methods, try to break down methods longer than 30 lines of code into smaller methods.

Code Quality Guideline: Avoid Long Methods